networkx/algorithms/connectivity/tests/test_edge_augmentation.py

```diff
@@ -297,7 +297,7 @@ def _augment_and_check(
             assert not isinstance(generator, list), "should always return an iter"
             aug_edges = []
             for edge in generator:
-                aug_edges.append(edge)
+                aug_edges.append(edge)  # noqa: PERF402
```

Comment:
I personally prefer keeping comments like this to a minimum if possible - part of my reticence of adding the linter for changes moving forward.

```suggestion
                aug_edges.append(edge)
```